### ISSUE TITLE
Can provide pretrained models?

### ISSUE THREAD

**Issue by sofiawu** [neouyghur/One-stage-Mask-based-Scene-Text-Eraser#2]
Could you provide your pretrained models for testing? thanks

**Comment by coolKeen** [comment]
Also want the pretrained model, please!!!! @neouyghur

**Comment by dinalzein** [comment]
same, I need the pre-trained models pls @neouyghur

**Comment by K024** [comment]
Found one in https://github.com/naver/garnet#pre-trained-models

**Comment by UsamaRehman001** [comment]
Any one of you guys trained the Ensnet on their dataset, I need help
@K024 @dinalzein  @coolKeen @sofiawu
